frontend/public/config.yml

```
@@ -438,9 +438,33 @@ collections:
           - { label: 'Intro do sveta', name: 'intro_body', widget: 'markdown' }
+        #Vlčie Skaly
+          - { label: 'Vlčie Skaly nadpis', name: 'vlcie_skaly_header', widget: 'string' }
+          - { label: 'Vlčie Skaly text', name: 'vlcie_skaly_body', widget: 'markdown' }
+          - { label: 'Vlčie Skaly obrázok', name: 'vlcie_skaly_image', widget: 'image' }
+        #guilds and groups
+          - { label: 'Cechy a skupiny nadpis', name: 'guilds_groups_header', widget: 'string' }
+          - { label: 'Cechy a spolky popis', name: 'guilds_groups_body', widget: 'markdown' }
+          - { label: 'Cechy nadpis', name: 'guilds_header', widget: 'string' }
+          - { label: 'Cechy popis', name: 'guilds_body', widget: 'markdown' }
+          - { label: 'Cechy (hráči sa vedia pridať)', name: 'guilds', widget: 'list', fields: [
+              { label: 'Názov', name: 'name', widget: 'string' },
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)